libs/service/relayer/src/handleProposalActivityMessage.ts

```diff
@@ -90,7 +90,7 @@ export const handleProposalActivityMessage = async (
 		}
 
 		if (!Object.keys(updatedData).length) return;
-		logger.info("Proposal #%d: 💬  update Discord [2/3]", proposalId);
+		logger.info("Proposal #%d: 💬 update Discord [2/3]", proposalId);
```

Comment:
Might want to update line `86` & `93` as well

Reply:
yeah emoji is a bit funny, some of them need 2 spaces, while others don't. They now looks aligned on when logging out to console,